app/mersea-front/src/components/Home.vue

```diff
@@ -49,13 +52,56 @@ export default {
         })
       )
       this.map.addControl(new mapboxgl.NavigationControl(), 'bottom-right')
+      this.map.on('load', async () => {
+        try {
+          let reports = await fetch(`${apiUrl}/reports`, {
+            method: 'get',
+            headers: new Headers({
+              Accept: 'application/geo+json',
+              'Content-Type': 'application/geo+json'
+            })
+          })
+          var geojson = await reports.json()
+        } catch (e) {
+          throw e
```

Comment:
Should add a better way to handle it? (A `TODO` should be fine until the PR for errors)

Reply:
Maybe use notification and such yes